src/app/pages/questions/components/question/question.component.html

```diff
@@ -106,4 +106,12 @@ <h2 class="taskformat">{{ question.field_label }}</h2>
       [type]="question.text_validation_type_or_show_slider_number"
     ></text-input>
   </div>
+
+  <div *ngIf="question.field_type === 'yesno'">
+    <radio-input
+      [responses]="[{'code':'1','label':'Yes'},{'code':'0','label':'No'}]"
+      (valueChange)="onValueChange($event)"
+    >
+    </radio-input>
+  </div>
```

Comment:
Why is this added? Isn't this just a specialisation of the radio field type? 
If using radio, we can just add the `{'code':'1','label':'Yes'},{'code':'0','label':'No'}` in question.select_choices_or_calculations when creating the definitions file.

Reply:
I guess that's also a possibility.  
I didn't want to change the definitions utility since it seemed to be very specific, whereas handling this in the app seemed to me as a broader scope. But I am also not sure if I am missing anything in the whole ecosystem that also needs to be changed to handle yesno fields properly.  
But yeah in the end a yesno is just a special radio with always these two options.

Reply:
Hi, Sebastian, I don't think any change to the utility of the definitions is needed if we use the existing radio field type in the definition file, this will be exactly same to the other radio questions but will have just 2 choices - Yes and No. I don't see the need to define a new `yesno` type.
Perhaps @mpgxvii can also comment here.

Reply:
Hmm but then you would have to make a hard requirement to just not use the yesno field type in redcap? Or does that requirement already exist and I didn't know about it?

Reply:
I don't think there is any hard requirement like that. So if you really need to use that type then it's ok to add (I commented to just point out that a similar result is already possible using the radio type).

BTW, you can also just add an `or` clause to the `radio` type for `yesno` as it is the same. Like on line 26 -

```html
  <div *ngIf="question.field_type === 'radio' || question.field_type === 'yesno'">
```

Reply:
Well, I could just redo the REDCap questionnaire, and reconvert to a new json (or just change in the json manually), but I thought that maybe supporting this out of the box might be a better approach, so we don't need such a hard requirement on new questionnaires that may come in the future.

Adding it by an `or` won't work I think, since the responses are not actually in the json, so they need to be filled in somewhere. [Here is an example yesno question](https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/blob/epilepsy-aRMT-study/questionnaires/seizure_diary/seizure_diary_armt.json#L65).

IMO it is fine like it is, If you think it is acceptable like this then you can go ahead and pull.

Reply:
Ok thanks. Did not know that it doesn't export the responses.